### PR TITLE
helloworld app.yaml outside of guestbook branches

### PR DESCRIPTION
**Pull request by angelafunk**
Create new folder so that go get can be used in app engine standard quickstart example

### CODE REVIEW

**Comment by broady** [comment]
I think we should have the actual hello world code here, too?

Something simple?

```
package main

import (
    "fmt"
    "net/http"
)

func main() {
    http.HandleFunc("/", handle)
    appengine.Main()
}

func handle(w http.ResponseWriter, r *http.Request) {
    fmt.Fprintln(w, "Hello, world!")
}
```

File: hello.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package main
```
Comment (broady) [inline hello.go]: This file's in the wrong directory. Should be in appengine/helloworld

**Comment by angelafunk** [comment]
Can we get this in the main branch?

**Comment by broady** [comment]
Done!

**Pull request merged**